catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/WorkspaceMetacardImpl.java

```diff
@@ -71,12 +73,26 @@ public void setMetacards(List<String> items) {
   }
 
   public List<String> getQueries() {
-    return AccessControlUtil.getValuesOrEmpty(this, WorkspaceConstants.WORKSPACE_QUERIES);
+    Attribute queryAttr = this.getAttribute(Associations.RELATED);
+    return queryAttr == null
+        ? new ArrayList<>()
```

Comment:
The only reason I'm not using it is because `Collections.emptyList()` is immutable. In this same class there's this method

```
  public void addQueryAssociation(String queryId) {
    List<String> queryIds = this.getQueries();
    queryIds.add(queryId);
    this.setQueries(queryIds);
  }
```

If I return `Collections.emptyList()` then I'll have to perform a check and create a new list which I was trying to avoid